helm-chart-sources/appscope/templates/configmap.yaml

```diff
@@ -0,0 +1,200 @@
+        tls:
+          enable: false
+          validateserver: true
+      {{- else if .Values.appscope.destinations.metrics | hasPrefix "tls://" }}
+      enable: true
+      transport:
+        type: tcp
+        host: {{ include "appscope.extractHost" (list .Values.appscope.destinations.metrics) }}
+        port: {{ include "appscope.extractPort" (list .Values.appscope.destinations.metrics) }}
+        tls:
+          enable: true
+          validateserver: true
+      {{- else if .Values.appscope.destinations.metrics | hasPrefix "unix://" }}
+      enable: true
+      transport:
+        type: unix
+        path: {{ include "appscope.extractUnixFilePath" (list .Values.appscope.destinations.metrics) }}
+      {{- end }}
+      format:
+        type: {{ .Values.appscope.metricFormat }}
+        verbosity: 4
+      watch:
+      - type: fs
+      - type: net
+      - type: http
+      - type: dns
+      - type: process
+      - type: statsd
+    event:
+      {{- if not .Values.appscope.destinations.events }}
+      enable: false
+      {{- else if eq .Values.appscope.destinations.events "edge" }}
+      enable: true
+      transport:
+        type: edge
+      {{- else if .Values.appscope.destinations.events | hasPrefix "file://" }}
+      enable: true
+      transport:
+        type: file
+        path: {{ include "appscope.extractUnixFilePath" (list .Values.appscope.destinations.events) }}
+        buffering: line
+      {{- else if .Values.appscope.destinations.events | hasPrefix "tcp://" }}
+      enable: true
+      transport:
+        type: tcp
+        host: {{ include "appscope.extractHost" (list .Values.appscope.destinations.events) }}
+        port: {{ include "appscope.extractPort" (list .Values.appscope.destinations.events) }}
+        tls:
+          enable: false
+          validateserver: true
+      {{- else if .Values.appscope.destinations.events | hasPrefix "udp://" }}
+      enable: true
+      transport:
+        type: udp
+        host: {{ include "appscope.extractHost" (list .Values.appscope.destinations.events) }}
+        port: {{ include "appscope.extractPort" (list .Values.appscope.destinations.events) }}
+        tls:
+          enable: false
+          validateserver: true
+      {{- else if .Values.appscope.destinations.events | hasPrefix "tls://" }}
+      enable: true
+      transport:
+        type: tcp
+        host: {{ include "appscope.extractHost" (list .Values.appscope.destinations.events) }}
+        port: {{ include "appscope.extractPort" (list .Values.appscope.destinations.events) }}
+        tls:
+          enable: true
+          validateserver: true
+      {{- else if .Values.appscope.destinations.events | hasPrefix "unix://" }}
+      enable: true
+      transport:
+        type: unix
+        path: {{ include "appscope.extractUnixFilePath" (list .Values.appscope.destinations.events) }}
+      {{- end }}
+      format:
+        type: ndjson
+    watch:
```

Comment:
This is rigid and doesn't allow customization. Can we move this to the `values.yaml` file?

Reply:
@bdalpe - you're referring to the ability to customize the `watch` section so a user can specify what they'd like monitored. Agreed. Let me define a sensible default, but it can be overridden by adding custom configuration via the watch file. Looking to reduce the burden of getting started by not requiring a ton of AppScope config file syntax knowledge. 

Does that make sense?

Reply:
Improved in the last commit.